src/util.h

```diff
@@ -29,6 +29,9 @@
 #include <boost/signals2/signal.hpp>
 #include <boost/thread/exceptions.hpp>
 
+// Application startup time (used for uptime calculation)
+static const int64_t nStartupTime = GetTime();
```

Comment:
Yes, sorry I wasn't clear - I didn't expect you to change the DEFAULT_LOG constants. I do however think you need to move the nStartupTime definition to avoid the following build warnings:

```
In file included from addrman.h:14:0,
                 from addrdb.cpp:8:
util.h:33:22: warning: ‘nStartupTime’ defined but not used [-Wunused-variable]
 static const int64_t nStartupTime = GetTime();
                      ^   
  CXX      libbitcoin_server_a-bloom.o
In file included from addrman.h:14:0,
                 from addrman.cpp:6:
util.h:33:22: warning: ‘nStartupTime’ defined but not used [-Wunused-variable]
 static const int64_t nStartupTime = GetTime();
                      ^   
  CXX      libbitcoin_server_a-blockencodings.o
 /bin/bash ./config.status
In file included from bitcoind.cpp:18:0:
util.h:33:22: warning: ‘nStartupTime’ defined but not used [-Wunused-variable]
 static const int64_t nStartupTime = GetTime();
                      ^
...(repeated many times)
```